engines/wintermute/base/base_game.cpp

```diff
@@ -3413,6 +3394,12 @@ bool BaseGame::externalCall(ScScript *script, ScStack *stack, ScStack *thisStack
 	}
 #endif
 
+#if 1
```

Comment:
Is this `#if` a leftover?

Reply:
It marks the block that should be removed to disable plugins. Similar `#if 1` marks enabling/disabling externals.

I thought of something similar to `#ifdef ENABLE_FOXTAIL` and `#ifdef ENABLE_HEROCRAFT`, so that build configuration can be switched between a vanilla engine for guys like https://jennibee.itch.io/dreamcat and extended engine for ScummVM and ResidualVM. However, I don't think that additional subengine is a good metaphor for this (since those extentions are from a pack of completely different games actually). 

What do you think I should do with those?

Reply:
I see a little benefit in disabling plugins in general or striving for "vanilla" engine. The game-specific changes are well-contained, and even if they change the engine behaviour, they are guarded by checks, that is normal.

We implement subengines only when there are _substantial_ code additions or different hardware requirements such as a bigger screen. This is not applicable to WM because it is a high-end engine anyway.

So, either remove those ifs or put a comment above them.

Reply:
That sounds reasonalble. Removed those `#if` at 899c569